src/test/server.c

```diff
@@ -125,6 +137,17 @@ void *pdispatch(void *data)
         printf("In server callback for msg #%ld\n", msg->hinfo.cookie);
     }
 
+    if (!authorize(msg->hinfo.target, msg->hinfo.instance))
```

Comment:
I implemented this in case of a service failure event, libqnio will talk to a directly to a remote VxHS server which should again authorize the client based on its X.509 cert. But come to think of it, before such an event a client should have successfully authorized itself atleast once with the local VxHS server. So maybe you are right, I don't need this extra authorization.